feature/xhr/aggregate/index.js

```diff
@@ -83,8 +85,8 @@ function storeXhr(params, metrics, startTime, endTime, type) {
     requestSize: metrics.txSize,
     responseSize: metrics.rxSize,
     type: type,
-    startTime: startTime,
-    endTime: endTime,
+    startTime: !useMetrics ? startTime : metrics.time,
+    endTime: !useMetrics ? endTime : metrics.time + metrics.duration,
```

Comment:
If for some reason after all of the event handlers have finished and  it still doesn't have endTime for building the event, the `storeXHR` method does have access to the metrics, which is what is used to build the start and endTime props anyway.  This can serve as a useful fallback.